distribution/ddf-common/src/main/resources/etc/definitions/core-attribute-injectors.json

```diff
@@ -1,5 +1,242 @@
 {
   "inject": [
+    {
+      "attribute": "associations.external"
+    },
+    {
+      "attribute": "metacard.associations.related"
+    },
+    {
+      "attribute": "metacard.associations.derived"
+    },
+    {
+      "attribute": "contact.contributor-address"
```

Comment:
❓ Just curious what defines these as core rather than having a separate injector file each set of attributes (like security)? Specifically things like `contact.*` and `media.*`.

Reply:
A separate injector for each seem excessive to me. I was thinking about renaming the file to something like ddf-catalog-taxonomy-injectors.json

Reply:
Okay, I think that would be a good rename 👍